public/components/management/cluster/node-list.tsx

```diff
@@ -44,7 +44,7 @@ export const NodeList = withErrorBoundary (class NodeList extends Component {
             },
             {
                 field: 'ip',
-                name: 'IP',
+                name: 'IP Address',
```

Comment:
**nitpick**: The other references use `IP address` with `address` in lowercase. Is there a reason to this case is different?